typesafe_conductr_cli/conduct_load.py

```diff
@@ -24,8 +24,8 @@ def load(args):
         bundleId = response_json['bundleId']
 
         print("Bundle loaded.")
-        print("Start bundle with: cli/conduct run {}".format(bundleId))
-        print("Unload bundle with: cli/conduct unload {}".format(bundleId))
-        print("Print ConductR info with: cli/conduct info")
+        print("Start bundle with: conduct run{} {}".format(args.cli_parameters, bundleId))
```

Comment:
Yes. There is a test for that: https://github.com/typesafehub/typesafe-conductr-cli/pull/1/files#diff-7a0c948b61473526c35ecd7260a0b766R70